crates/iota/src/validator_commands.rs

```diff
@@ -1241,9 +1244,9 @@ async fn update_metadata(
             .await
         }
         MetadataUpdate::PrimaryAddress { primary_address } => {
-            primary_address.to_anemo_address().map_err(|_| {
```

Comment:
the former `to_anemo_address` would check a more explicit format, no? Should we really change that to `is_loosely_valid_tcp_addr`?

Reply:
we changed the code that all primary addresses are now checked via `is_loosely_valid_tcp_addr` because they are already valid multiaddresses.